ci/jenkins/test.sh

```diff
@@ -839,6 +853,19 @@ function run_conformance {
     kubectl rollout status daemonset/antrea-agent -n kube-system
 
     set +e
+    # Conformance DNS test cases fail in ipv6 Kind cluster because here CoreDNS pods are IPV6 pods 
+    # and it can't reach IPV4 upstream nameserver.
+    # Note:Default CoreDNS ConfigMap uses /etc/resolv.conf to get upstream nameservers listed in resolv.conf file on the host,
+    # the upstream nameservers are accessed by IPV4, when single stack IPV6 pods try to connect ipv4 nameserver it return servfail,
+    # with the new ConfigMap we will forward DNS query to "vmware.com:53" and it will return 'NXDOMAIN' so that client keep searching.
+    # For more detail: https://github.com/kubernetes/kubernetes/issues/94794 
+    if [[ $IP_MODE == "ipv6" && "$TESTCASE" =~ "conformance" && $TESTBED_TYPE == "kind" && $JENKINS_LOCATION == "lab" ]]; then
+        # Override the CoreDNS ConfigMap with new one.
+        kubectl apply -f ./ci/coredns-configmap.yaml
```

Comment:
Will you check and make sure the kind related steps are running with the same context?

Reply:
yes